### PR TITLE
Fixed Mobile version styling for Meraki team, Volunteer with us, Our …

### PR DESCRIPTION
**Pull request by amansharmma**
The styling part for the mobile version of the Meraki team page, Volunteer with us & Our partner is updated here.

### CODE REVIEW

File: src/components/VolunteerAutomation/index.js [hunk 1]
```diff
@@ -194,24 +196,36 @@ function VolunteerAutomation() {
 
               <Box className={classes.displayIcon}>
                 <ArrowRightAltIcon />
-                <Typography className={classes.TextContent}>
+                <Typography
+                  variant="subtitle1"
+                  className={classes.TextContent}
+                  Typography
```
Comment (Poonam-Singh-Bagh) [inline src/components/VolunteerAutomation/index.js]: Why do you need to use `Typography` attribute. Remove it.

File: src/components/VolunteerAutomation/index.js [hunk 2]
```diff
@@ -194,24 +196,36 @@ function VolunteerAutomation() {
...
                 </Typography>
               </Box>
 
               <Box className={classes.displayIcon}>
                 <ArrowRightAltIcon />
-                <Typography className={classes.TextContent}>
+                <Typography
+                  variant="subtitle1"
+                  className={classes.TextContent}
+                  Typography
```
Comment (Poonam-Singh-Bagh) [inline src/components/VolunteerAutomation/index.js]: Remove `Typography` attribute

**Review by Poonam-Singh-Bagh — CHANGES_REQUESTED** [review]

File: src/components/VolunteerAutomation/index.js [hunk 3]
```diff
@@ -194,24 +196,36 @@ function VolunteerAutomation() {
...
                   Project Management
                 </Typography>
               </Box>
               <Box className={classes.displayIcon}>
                 <ArrowRightAltIcon />
-                <Typography className={classes.TextContent}>
+                <Typography
+                  variant="subtitle1"
+                  className={classes.TextContent}
+                  Typography
```
Comment (Poonam-Singh-Bagh) [inline src/components/VolunteerAutomation/index.js]: Remove `Typography` attribute

File: src/pages/OurPartner/index.js [hunk 4]
```diff
@@ -57,12 +67,13 @@ const OurPartner = () => {
               target="_blank"
             >
               <Button
-                Width
+                // Width
```
Comment (Poonam-Singh-Bagh) [inline src/pages/OurPartner/index.js]: Remove it.

**Review by Poonam-Singh-Bagh — CHANGES_REQUESTED** [review]

**Comment by amansharmma** [comment]
Added Some more styling and functionality fixes.
Fixed card part of it.
Screenshot from 2022-09-03 17-50-08

Text contact area.
Screenshot from 2022-09-03 17-49-46

Card full-width.
Screenshot from 2022-09-03 17-48-42

Removed link & cards for it.
Screenshot from 2022-09-03 17-48-06

**Pull request merged**